docs/examples.rst

```diff
@@ -0,0 +1,174 @@
+
+    # Run job every day at specific timestamp
+    schedule.every().day.at("10:30").do(job)
+    schedule.every().day.at("10:30").do(job)
+
+    # Run job on a specific day of the week
+    schedule.every().monday.do(job)
+    schedule.every().wednesday.at("13:15").do(job)
+    schedule.every().minute.at(":17").do(job)
+
+    while True:
+        # run_pending
+        schedule.run_pending()
+        time.sleep(1)
+
+Pass arguments to a job
+~~~~~~~~~~~~~~~~~~~~~~~
+
+``do()`` passes extra arguments to the job function
+
+.. code-block:: python
+
+    import schedule
+
+    def greet(name):
+        print('Hello', name)
+
+    schedule.every(2).seconds.do(greet, name='Alice')
+    schedule.every(4).seconds.do(greet, name='Bob')
+
+
+Cancel a job
+~~~~~~~~~~~~
+To remove a job from the scheduler, use the ``schedule.cancel_job(job)`` method
+
+.. code-block:: python
+
+    import schedule
+
+    def some_task():
+        print('Hello world')
+
+    job = schedule.every().day.at('22:30').do(some_task)
+    shcedule.cancel_job(job)
+
+
+Run a job once
+~~~~~~~~~~~~~~
+
+Return ``schedule.CancelJob`` from a job to remove it from the scheduler.
+
+.. code-block:: python
+
+    import schedule
+    import time
+
+    def job_that_executes_once():
+        # Do some work that only needs to happen once...
+        return schedule.CancelJob
+
+    schedule.every().day.at('22:30').do(job_that_executes_once)
+
+    while True:
+        schedule.run_pending()
+        time.sleep(1)
+
+
+Cancel all jobs
+~~~~~~~~~~~~~~~
+To remove all jobs from the scheduler, use ``schedule.clear()``
+
+.. code-block:: python
+
+    import schedule
+
+    def greet(name):
+        print('Hello {}'.format(name))
+
+    schedule.every().second.do(greet)
+
+    schedule.clear()
+
+
+Cancel several jobs at once
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can cancel the scheduling of a group of jobs selecting them by a unique identifier.
+
+.. code-block:: python
+
+    import schedule
+
+    def greet(name):
+        print('Hello {}'.format(name))
+
+    schedule.every().day.do(greet, 'Andrea').tag('daily-tasks', 'friend')
+    schedule.every().hour.do(greet, 'John').tag('hourly-tasks', 'friend')
+    schedule.every().hour.do(greet, 'Monica').tag('hourly-tasks', 'customer')
+    schedule.every().day.do(greet, 'Derek').tag('daily-tasks', 'guest')
+
+    schedule.clear('daily-tasks')
+
+Will prevent every job tagged as ``daily-tasks`` from running again.
+
+Run a job at random intervals
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    def my_job():
+        print('Foo')
+
+    # Run every 5 to 10 seconds.
+    schedule.every(5).to(10).seconds.do(my_job)
+
+``every(A).to(B).seconds`` executes the job function every N seconds such that A <= N <= B.
+
+
+Run all jobs now, regardless of their scheduling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To run all jobs regardless if they are scheduled to run or not, use ``schedule.run_all()``
```

Comment:
Does running all jobs affect when the jobs will run again or will they keep their original schedule?
I assume it doesn't, but it can make it clearer when added.
For example: schedule.every(5)....

Reply:
Good one! I've added some clarification on that.